src/Mapbender/CoreBundle/Element/Type/RulerAdminType.php

```diff
@@ -30,18 +31,40 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ))
             ->add('type', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType',
                 array(
-                'required' => true,
-                'choices' => array(
-                    "line" => "line",
-                    "area" => "area",
-                ),
-                'choices_as_values' => true,
-            ))
+                    'required' => true,
+                    'choices' => array(
+                        "line" => "line",
+                        "area" => "area",
+                    ),
+                    'choices_as_values' => true,
+                ))
             ->add('immediate', 'Symfony\Component\Form\Extension\Core\Type\CheckboxType', array(
                 'required' => false,
                 'label' => 'Immediate',
             ))
-        ;
+            ->add('strokeWidth', 'Symfony\Component\Form\Extension\Core\Type\IntegerType',
+                array('required' => false,
+                    'label' => 'Str. Width',
+                    'attr' => ['min' => 1, 'max' => 8],
+                    'data' => 1))
+            ->add('strokeOpacity', 'Symfony\Component\Form\Extension\Core\Type\RangeType',
+                array('required' => false, 'label' => 'Str. Opacity', 'attr' => [
```

Comment:
Please don't mix short and long `Array` syntax

Reply:
mxing of array syntax is resolved,  but decided against 'Line' and 'Polygon' as values in order to be minimally invasive.